newsroom/news_api/news/syndicate/__init__.py

```diff
@@ -0,0 +1,72 @@
+import superdesk
+import logging
+import json
+from flask import request, make_response, jsonify
+from eve.methods.get import get_internal
+from .error_handlers import process_error_response
+from .auth import authenticate
+from .syndicate_handlers import FORMAT_HANDLERS, FEED_GENERATORS as FORMAT_HANDLERS_INIT
+from .resource import NewsAPISyndicateResource
+from .service import NewsAPISyndicateService
+from werkzeug.routing import BaseConverter
+
+blueprint = superdesk.Blueprint('syndicate', __name__)
+
+logger = logging.getLogger(__name__)
+
+
+class RegExConverter(BaseConverter):
+    def __init__(self, map, regex='[^/]+'):
+        super().__init__(map)
+        self.regex = regex
+
+
+def get_feed(syndicate_formatter, token=None):
+
+    def generate_feed(syndicate_formatter):
+        response = get_internal('news/search')
```

Comment:
This code could be placed directly in the blueprint view, without the need for `get_feed` nor `generate_feed` functions